config/gov_delivery.yml

```diff
@@ -1,7 +1,7 @@
 development:
   username: govdelivery@example.com
   password: nottherealpassword
-  account_code: UKGOVUK
```

Comment:
Let talk about this as I feel it should be UKGOVUK for development, but that means we need to support different values in staging/production so might need to add a new confirm variable somewhere.